ocs_ci/ocs/resources/pod.py

```diff
@@ -480,6 +481,8 @@ def check_file_existence(pod_obj, file_path):
     Returns:
         bool: True if the file exist, False otherwise
     """
+    if fedora_dc:
+        pod_obj.install_packages("findutils")
```

Comment:
I think we should check for the package regardless of the pod type and install it if not found.
Then we don't need `fedora_dc` parameter.

Reply:
Done